cmds/task/actors.go

```diff
@@ -149,6 +210,38 @@ func runComplete(credentials *tcclient.Credentials, args []string, out io.Writer
 		return fmt.Errorf("could not complete the task %s: %v", taskID, err)
 	}
 
+
```

Comment:
It looks like this performs the completion, *then* asks for confirmation, and if "y" runs the completion *agian*?  I don't think that's right!